Sources/Widgets/Core/WidgetManager/example/index.js

```diff
@@ -39,7 +53,24 @@ const WIDGET_BUILDERS = {
     const instance = widgetManager.addWidget(
       vtkPolyLineWidget.newInstance({
         label: 'Closed Polyline',
-      })
+      }),
+      null,
+      {
+        coincidentTopologyParameters: {
+          Point: {
+            factor: -1.0,
+            offset: -1.0,
+          },
+          Line: {
+            factor: -1.5,
+            offset: -1.5,
+          },
+          Polygon: {
+            factor: -2.0,
+            offset: -2.0,
+          },
+        },
+      }
```

Comment:
Since this is the same on line 34 and here, could you define this as a static object and pass it to both `instance.setCoincidentTopologyParameters` and `widgetManager.addWidget`?

Reply:
I just realized that was in the example and no the actual class, so doesn't matter as such.